### PR TITLE
APT-4445: ASan is disabled by default in Debug builds, but can be enabled (used in CI)

### PR DESCRIPTION
**Pull request by iurii-provizio**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -279,9 +308,10 @@ add_library(
   src/core.c)
 target_include_directories(provizio_radar_api_core PUBLIC include)
 target_include_directories(provizio_radar_api_core SYSTEM
-                           PRIVATE "${LINMATH_HEADER_DIR}")
+  PRIVATE "${LINMATH_HEADER_DIR}")
 set_property(TARGET provizio_radar_api_core PROPERTY C_STANDARD 99) # As defined
-                                                                    # by MISRA
+
+# by MISRA
```
Comment (ifabek) [inline CMakeLists.txt]: Shouldn't "As defined " and "by MISRA" be one comment? Why split into 3 lines?

**Review by ifabek — DISMISSED** [review]
Looks good to me.

File: CMakeLists.txt (hunk 1 above)
Comment (iurii-provizio) [reply CMakeLists.txt]: @ifabek Ah, it's the auto-formatter. Thank you, will fix it now

**Review by ifabek — APPROVED** [review]
It is ready.

**Pull request merged**